rust-src/concordium_base/src/bulletproofs/range_proof.rs

```diff
@@ -13,7 +13,7 @@ use rand::*;
 use std::iter::once;
 
 /// Bulletproof style range proof
-#[derive(Clone, Serialize, SerdeBase16Serialize, Debug)]
+#[derive(Clone, Eq, PartialEq, Serialize, SerdeBase16Serialize, Debug)]
```

Comment:
It is not trivial what `Eq` means for something like a proof, it would be good to document what to expect from it